src/components/results/pccmin/pcc-min-result-tab.tsx

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { Box, LinearProgress, Tab, Tabs } from '@mui/material';
+import { FunctionComponent, useCallback, useEffect, useMemo, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { ComputationReportViewer } from '../common/computation-report-viewer';
+
+import { useSelector } from 'react-redux';
+import { AppState } from '../../../redux/reducer';
+import { ComputingType } from '@gridsuite/commons-ui';
+import { RunningStatus } from '../../utils/running-status';
+import { useOpenLoaderShortWait } from '../../dialogs/commons/handle-loader';
+import { RESULTS_LOADING_DELAY } from '../../network/constants';
+import GlobalFilterSelector from '../common/global-filter/global-filter-selector';
+import { EQUIPMENT_TYPES } from '../../utils/equipment-types';
+import useGlobalFilters, { isGlobalFilterParameter } from '../common/global-filter/use-global-filters';
+import { useGlobalFilterOptions } from '../common/global-filter/use-global-filter-options';
+import { PccMinResultTabProps } from './pcc-min-result.type';
+import { PccMinResult } from './pcc-min-result';
+
+export const PccMinResultTab: FunctionComponent<PccMinResultTabProps> = ({
+    studyUuid,
+    nodeUuid,
+    currentRootNetworkUuid,
+}) => {
+    const [resultOrLogIndex, setResultOrLogIndex] = useState(0);
+
+    const pccMinStatus = useSelector((state: AppState) => state.computingStatus[ComputingType.PCC_MIN]);
+
+    const RESULTS_TAB_INDEX = 0;
+    const LOGS_TAB_INDEX = 1;
+
+    const { globalFilters, handleGlobalFilterChange } = useGlobalFilters();
+    const { countriesFilter, voltageLevelsFilter, propertiesFilter } = useGlobalFilterOptions();
+
+    const handleSubTabChange = useCallback(
+        (event: React.SyntheticEvent, newIndex: number) => {
+            setResultOrLogIndex(newIndex);
+        },
+        [setResultOrLogIndex]
```

Comment:
```suggestion
        []
```
no need to add useState setter in dependency